docs/docusaurus/docs/core/set_up_a_gx_environment/_create_a_data_context/cloud_data_context.py

```diff
@@ -20,3 +20,5 @@
 print(context)
 # </snippet>
 # </snippet>
+
+assert type(context).__name__ == "CloudDataContext"
```

Comment:
not that it matters, but I'd usually expect this to just be `assert isinstance(context, CloudDataContext)`